compiler/ztests/sql/readme-query.yaml

```
@@ -12,7 +12,7 @@ inputs:
       | FORK
         (
           FROM eval(f'{user}.sup')
-          SELECT VALUE {user:login,created_at:created_at::time}
+          SELECT login as user, created_at::time as created_at
```

Nit: All other SQL reserved words here are uppercase.

```suggestion
          SELECT login AS user, created_at::time AS created_at
```